packages/common/src/streamopts.ts

```diff
@@ -68,6 +69,14 @@ export interface LoadOpts extends SyncOpts, PinningOpts {
    * Load a previous version of the stream based on unix timestamp
    */
   atTime?: number
+  /**
+   * Provide the genesis content so it does not need to fetched from IPFS daemon
+   */
+  genesis?: Record<string, any>
```

Comment:
you shouldn't need to make any changes to LoadOpts.  Any fields added to LoadOpts would need to be supported in the primary loadStream API, and be able to be serialized for use in the http client